pkg/cli/cmds/server.go

```diff
@@ -212,6 +213,11 @@ var ServerFlags = []cli.Flag{
 		Destination: &ServerConfig.FlannelBackend,
 		Value:       "vxlan",
 	},
+	&cli.BoolFlag{
+		Name:        "multi-cluster-cidr",
+		Usage:       "(networking) Enable multiClusterCIDR (v1.26 Alpha feature)",
```

Comment:
The convention is to put `experimental` in the flag group portion of the usage string.
```suggestion
		Usage:       "(experimental/networking) Enable multiClusterCIDR",
```